### PR TITLE
New package: KelVPN.KelVPN version 7.7-3

### PR DESCRIPTION
**Pull request by Exorcism0666**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] This PR only modifies one (1) manifest
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.5 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.5.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

---

 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/135291)

### CODE REVIEW

**Comment by Exorcism0666** [comment]
image

**Comment by Exorcism0666** [comment]
## For moderators

This account is automated by Github Actions and the source code was created by CoolPlayLin. If you have any questions about any pull request, don't hesitate to ping @Exorcism0666, I'll get a notification.
>[!important]
>Please carefully review these Pull Request before merging. If it is a Pull Request for removing incorrect content and the URLs are issue, free upon manual checking, please close this Pull Request directly. (It is best to inform Exorcism0666 of the closure.)

良い一日をお過ごしください！

**Comment by stephengillie** [comment]
Automatic Validation ended with:
> 

(Automated response - build 628.)

File: manifests/k/KelVPN/KelVPN/7.7-3/KelVPN.KelVPN.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.5.7.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.5.0.schema.json
+
+PackageIdentifier: KelVPN.KelVPN
+PackageVersion: 7.7-3
+InstallerType: nullsoft
+Installers:
+- InstallerUrl: https://pub.kelvpn.com/windows/rieman-test_new/KelVPN-7.7-3-installer.exe
+  Architecture: x86
```
Comment (stephengillie) [inline manifests/k/KelVPN/KelVPN/7.7-3/KelVPN.KelVPN.installer.yaml:9]:
```suggestion
Architecture: x64
```

**Comment by stephengillie** [comment]
Automatic Validation ended with:
> No errors to post.

(Automated response - build 675.)

**Comment by stephengillie** [comment]
Manual Validation ended with: 
image

Is an installer switch available to bypass this message and install the driver? Unfortunately, this is rare.

**Pull request closed without merging**